lib/image_render.dart

```diff
@@ -148,7 +147,7 @@ ImageRender networkImageRender({
         initialData: context.parser.cachedImageSizes[src],
         builder: (BuildContext buildContext, AsyncSnapshot<Size> snapshot) {
           if (snapshot.hasData) {
-            return Container(
+            var imageWidget = Container(
```

Comment:
I don't think splitting this into a variable and returning it later made any difference in the code, unless I missed something. What I see is 

```
var widget = Container(...);
return widget;
```

I believe we should just leave it to `return Container(...)`.

Reply:
Whoops! I did have another condition in there that would return something else but ended up removing it. Fixed.